### PR TITLE
Subscriber Badge Plugin and User Tags bug

### PR DESCRIPTION
**Pull request by okbel**
## What does this PR do?
- Adds subscriber Badge 
- Solves bug - https://www.pivotaltracker.com/story/show/151588649

## How do I test this PR?
- Add `"talk-plugin-subscriber"` to your plugins.json

### CODE REVIEW

File: plugins/talk-plugin-subscriber/client/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "parser": "babel-eslint",
+  "plugins": [
+    "react"
+  ],
+  "rules": {
+    "react/jsx-uses-react": "error",
+    "react/jsx-uses-vars": "error",
+    "no-console": ["warn", { "allow": ["warn", "error"] }]
+  }
+}
```
Comment (wyattjoh) [inline plugins/talk-plugin-subscriber/client/.eslintrc.json]: Should use the new eslint-preset. Check out the other plugin's file.

**Review by wyattjoh — CHANGES_REQUESTED** [review]

File: client/coral-embed-stream/src/graphql/index.js [hunk 2]
```diff
@@ -129,7 +134,19 @@ export default {
...
               id: auth.user.id,
-              username: auth.user.username
+              username: auth.user.username,
+              tags: tags.map((tag) => ({
+                tag: {
+                  name: tag,
+                  created_at: new Date().toISOString(),
+                  __typename: 'Tag'
+                },
+                assigned_by: {
```
Comment (alinhle) [inline client/coral-embed-stream/src/graphql/index.js:144]: This field I think was not requested in the query.

File: plugins/talk-plugin-subscriber/client/.eslintrc.json (hunk 1 above)
Comment (okbel) [reply plugins/talk-plugin-subscriber/client/.eslintrc.json]: 👍

File: client/coral-embed-stream/src/graphql/index.js (hunk 2 above)
Comment (okbel) [reply client/coral-embed-stream/src/graphql/index.js:144]: 👍

Comment (okbel) [reply client/coral-embed-stream/src/graphql/index.js:144]: But keep in mind that if I remove that field and a plugin request that it won't work.

**Review by wyattjoh — APPROVED** [review]

**Pull request merged**